### PR TITLE
fix: move resize handle same row as multiline input adornment

### PR DESCRIPTION
**Pull request by augustobmoura**
[FX-3144]

### Description

Move the resize handle for multiline Input to the bottom bar.

### How to test

- Check demo and Happo

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| image | image |

### Development checks

- [x] Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
- [x] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- [x] Make sure that additions and changes on design follow [Toptal's BASE design](https://design.toptal.net/), and it's been already discussed with designers at #-base-core
- [x] Annotate all `props` in component with documentation
- [x] Create `examples` for component
- [x] Ensure that deployed demo has expected results and good examples
- [x] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).
- [x] Self reviewed
- [x] Covered with tests

> All **_development checks_** should be done and set checked to pass the
> **GitHub Bot: TODOLess** action

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run package:alpha-release` - Release alpha version
- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[FX-3144]: https://toptal-core.atlassian.net/browse/FX-3144?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ozgurkececioglu** [comment]
Screenshot 2022-11-07 at 13 16 13
Screenshot 2022-11-07 at 13 16 24
Screenshot 2022-11-07 at 13 17 06

Nice CSS work 👏 I have tested the PoC in chrome, FF and safari, looks awesome 💯 

I'm still thinking like we can implement our own js solution instead of using `resize` CSS attribute so it can be consistent across all browsers 🤔 but I think it is Matt's responsibility to continue with this or request for more.

**Review by ozgurkececioglu — COMMENTED** [review]
for 5 rows, the height of the input should contain 5-row height but it is much more now

| before | after |
| ----- | ----- |
|Screenshot 2022-11-08 at 10 04 20|Screenshot 2022-11-08 at 10 04 17|

**Review by TomasSlama — APPROVED** [review]
Tested in temploy, good job Augusto 👍

File: packages/picasso/src/Input/Input.tsx [hunk 1]
```diff
@@ -282,6 +282,13 @@ export const Input = forwardRef<HTMLInputElement, Props>(function Input(
           }),
         }),
         input: cx(classes.input, {
+          [classes.inputMultiline]: multiline,
+          [classes.inputMultilineWithAdornment]: hasMultilineAdornment({
```
Comment (ozgurkececioglu) [inline packages/picasso/src/Input/Input.tsx:286]: we can calculate this `hasMultilineAdornment` once before `return`, this newly added is the second one.

**Review by OndrejTuma — COMMENTED** [review]
Two things I noticed that we should probably take care of:
1) there should be some minimal height in multiline input, otherwise I'm allowed to resize it to zero height
2) the line with character count should probably have background color so that when resizing bunch of text there is no overlap

https://user-images.githubusercontent.com/22159416/200527804-cf80e6ad-b51b-4ac2-bb16-72d3330900da.mov

Otherwise its looking good 🎉

**Comment by LashaJini** [comment]
> Two things I noticed that we should probably take care of:
> 
> 1. there should be some minimal height in multiline input, otherwise I'm allowed to resize it to zero height
> 2. the line with character count should probably have background color so that when resizing bunch of text there is no overlap
> 
>  testing-multiline-resize.mov 
> Otherwise its looking good tada

@augustobmoura Solution to similar issue is used in `RichTexEditor` resize handler in https://github.com/toptal/picasso/blob/master/packages/picasso/src/RichTextEditorCounter/styles.ts#L5 and https://github.com/toptal/picasso/blob/master/packages/picasso/src/RichTextEditor/styles.ts#L6

File: packages/picasso/src/Input/styles.ts [hunk 2]
```diff
@@ -8,18 +9,44 @@ import '../InputAdornment/styles'
...
+      '&$inputLarge': {
+        // 1rem line + top 0.625rem + bottom 2.125rem padding
+        minHeight: '3.75rem',
+        paddingBottom: '2.125rem',
+      },
     },
     inputMultilineResizable: {
       resize: 'vertical',
     },
+    inputLarge: {},
```
Comment (LashaJini) [inline packages/picasso/src/Input/styles.ts:51]: No design for large input or there is something else?

**Review by mkrl — APPROVED** [review]
Very nice. I have tested the current implementation and I love it.
One idea I had in mind is that we might want to try covering a resize handle with a non-transparent image with the consistent cross-browser handle we want, and then just give it `pointer-events: none`.
But that's only we we absolutely want that super consistent interface everywhere. I think it looks good enough as it is

**Comment by augustobmoura** [comment]
Closing because we decided on trying a new solution, the pure CSS solution was becoming very hacky and fragile with a lot of edge-cases. We are moving into either implementing our own resizing feature or using some from another library (like [react-resizable](https://github.com/react-grid-layout/react-resizable))

**Pull request closed without merging**